chokuretsu/patch.html

```diff
@@ -27,133 +27,157 @@
-    <div id="patcher-menu">
-        <div class="patcher-block">
-            <h3>Select ROM</h3>
-            <div id="patcher-file-drop" class="enabled">
-                <input type="file" id="input-file-rom" class="enabled" accept=".nds"
-                       ondragenter="this.classList.add('patcher-file-dragging');"
-                       ondragleave="this.classList.remove('patcher-file-dragging');"/>
+<main id="column-with-sidebar">
+    <div class="column-with-sidebar-body">
+        <div class="one-column-text">
+            <div id="game-logo-container">
+                <object class="image-graphic logo drop-shadow" width="100%" data="assets/game-logo.svg"
+                        type="image/svg+xml">
+                    <img src="assets/game-logo.png" alt="The Series of Haruhi Suzumiya logo"/>
+                </object>
             </div>
-        </div>
-
-        <div class="patcher-block">
-            <h3>Integrity checks</h3>
-            <ul style="text-align: left">
-                <li>CRC32: <code id="crc32"></code></li>
-                <li>MD5: <code id="md5"></code></li>
-                <li>SHA-1: <code id="sha1"></code></li>
-            </ul>
-        </div>
 
-        <div class="patcher-block">
-            <h3 id="select-patch-version">Select version</h3>
-            <div id="row-file-patch">
-                <input type="file" id="input-file-patch" accept=".xdelta"/>
+            <h1>Patch ROM</h1>
+            <div id="patcher-notice" class="notice-box hidden"></div>
+            <p>Please select your dumped <i>Suzumiya Haruhi no Chokuretsu</i> <code>.nds</code> ROM file (see the <a
+                    href="./guide/" target="_blank">setup guide</a> for how to get this), choose the options you'd
+                like to use, then click "Patch ROM" to apply the patch.</p>
+
+            <!-- Options menu -->
+            <div id="patcher-menu">
+                <div id="patcher-options">
+                    <h3>Options</h3>
+                    <table id="patcher-options-table">
+                        <tr>
+                            <td colspan="2">
+                                <img src="./assets/opening-subtitles.png"
+                                     alt="Screenshot of the opening movie with English subtitles and karaoke track"
+                                     class="patcher-option-image"/>
+                                <b>OP/ED subtitles</b><br/>
+                                Configure whether the opening movie and end credits should be subtitled in English with
+                                Japanese karaoke typesetting.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="patcher-option-radio">
+                                <div>
+                                    <label>
+                                        <input type="radio" name="op-ed-subtitling" value="subbedoped" checked/>
```

Comment:
super nitpicky nit so if the answer is "no" that's totally fine: is there a way we can pull these radio buttons out so that they're at the same Y-coordinate? rn the ones with "recommended" are higher up than the other ones bc of the line break.

Reply:
also yes i realize this is my fault lol

Reply:
![image](https://user-images.githubusercontent.com/31187453/161083960-b5d1469d-f753-4f46-af58-4300000e9a4f.png)
How's this?

Reply:
👍